press/press/doctype/site_update/site_update.py

```diff
@@ -466,7 +464,7 @@ def has_pending_updates(self):
 			"Site Update",
 			{
 				"site": self.site,
-				"status": ("in", ("Pending", "Running", "Failure", "Scheduled", "Recovering")),
```

Comment:
not sure about this but by just reading the code this didn't make sense to me

maybe this means something else?

Reply:
prolly shouldn't touch unrelated stuff :P